dcc-storage-server/pom.xml

```diff
@@ -156,6 +156,21 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
           </execution>
         </executions>
       </plugin>
+      
+      <!-- Jar -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <mainClass>org.icgc.dcc.storage.server.ServerMain</mainClass>

```

Comment:
Actually, I may be wrong here. The main point is that the entry point is always `org.springframework.boot.loader.JarLauncher`